zest/releaser/tests/hg.txt

```diff
@@ -73,9 +73,9 @@ Originally there are no tags (the default 'tip' tag is filtered out):
 
 Create a tag and it will show up:
 
-    >>> cmd = checkout.cmd_create_tag('0.1')
+    >>> cmd = checkout.cmd_create_tag('0.1', 'Create tag 0.1')
```

Comment:
Same here. "Tagging" is fine, we don't need to test customized messages here.

Reply:
Dito.

Reply:
create => creating and I'm happy.